heartofthealien/heartofthealien-1.2.2.recipe

```diff
@@ -5,10 +5,10 @@ COPYRIGHT="2004 Gil Megidish"
 LICENSE="GNU GPL v2"
 REVISION="1"
 srcGitRev="8f1f525fcf50c4610302e870c85625c8ea2f33eb"
-SOURCE_URI="https://github.com/ytmytm/beos-heartofthealien/archive/$ srcGitRev.tar.gz"
-SOURCE_FILENAME="heartofthealien- $ PORTVERSION ~ $ srcGitRev.tar.gz"
-SOURCE_DIR="BeOS-heartofthealien- $ srcGitRev"
-PATCHES="heartofthealien- $ portVersion.patch"
+SOURCE_URI="https://github.com/ytmytm/beos-heartofthealien/archive/$srcGitRev.tar.gz"
+SOURCE_FILENAME="heartofthealien-$PORTVERSION~$srcGitRev.tar.gz"
```

Comment:
Should be
```
SOURCE_FILENAME="heartofthealien-$portVersion~$srcGitRev.tar.gz"
```
like I wrote in my first review.
upper-case and lower-case have really different meanings. You can't change the case without a reason ;-)